node/index.mjs

```diff
@@ -0,0 +1,32 @@
+let parts = [process.platform, process.arch];
+if (process.platform === 'linux') {
+  const {MUSL, family} = await import('detect-libc');
+  if (family === MUSL) {
+    parts.push('musl');
+  } else if (process.arch === 'arm') {
+    parts.push('gnueabihf');
+  } else {
+    parts.push('gnu');
+  }
+} else if (process.platform === 'win32') {
+  parts.push('msvc');
+}
+
+let default_export;
+if (process.env.CSS_TRANSFORMER_WASM) {
+  default_export = await import('./pkg/parcel_css_node.js');
+} else {
+  const {createRequire} = await import('module');
+  const require = createRequire(import.meta.url);
+
+  try {
+    default_export = require(`@parcel/css-${parts.join('-')}`);
+  } catch (err) {
+    default_export = require(`../parcel-css.${parts.join('-')}.node`);
+  }
+}
+
+export default default_export;
```

Comment:
Maybe some of the above logic could be deduplicated with the `index.js` file? 

```js
import {createRequire} from 'module';
const require = createRequire(import.meta.url);

export default require('./index.js');
```

Reply:
What about the reverse? Generating the old commonjs file from the mjs file?

Reply:
I guess it could be, but it would just be CommonJS to load the native module regardless.